community/sway/etc/sway/modes/default

```diff
@@ -75,16 +75,16 @@ $bindsym $mod+9 workspace $ws9
 $bindsym $mod+0 workspace $ws10
 
 ## Action // Move focussed window to workspace // $mod + Shift + [number] ##
-$bindsym $mod+Shift+1 move container to workspace $ws1
-$bindsym $mod+Shift+2 move container to workspace $ws2
-$bindsym $mod+Shift+3 move container to workspace $ws3
-$bindsym $mod+Shift+4 move container to workspace $ws4
-$bindsym $mod+Shift+5 move container to workspace $ws5
-$bindsym $mod+Shift+6 move container to workspace $ws6
-$bindsym $mod+Shift+7 move container to workspace $ws7
-$bindsym $mod+Shift+8 move container to workspace $ws8
-$bindsym $mod+Shift+9 move container to workspace $ws9
-$bindsym $mod+Shift+0 move container to workspace $ws10
+$bindsym $mod+Shift+1 move container to workspace $ws1, exec [ $focus_after_move == 'true' ] && swaymsg workspace $ws1
```

Comment:
Can you elaborate on this one? :D
I don't quite get this trick. Okay, so in case $focus_after_move is true, we will `exec swaymsg workspace $ws1-9` after moving the window  (which, as I was writing I understood, won't be doomed by Zsh because $ws1-9 will get substituted by "number 1-9". Alright, I get it, but we are executing focus_after_move comparison every time we move something from workspace to workspace, which is kinda weird, this setting is expected to change on Sway restart, there's no need for it to be so dynamic.
While the solution is elegant in the code, it's suboptimal in runtime. If you are fine with that then OK, it will definitely work, but I thought something as a static import will be a bit more practical 🤷🏻‍♂️

Reply:
I have no non-dynamic idea though. And I expect this to still be very lightweight. Its an sh string comparison after all.

Reply:
My idea is the same as it was in the first comment message, and the same way it works with themes: define $focus_after_move in /etc/sway/definitions (focusing|non_focusing, for example). Make two files, define mod+shift+1-0 hotkeys here, one switching the focus and the other not. Of course deleting these definitions from the `default` mode, and including these files at the end of `default` mode file. Could be rewritten by user in `~/.config/sway/definitions`. I'm actually thinking to make a PR to move everything theme-based to output settings, mainly because it will allow to change $theme variable from ~/.config/sway/definitions. Though I still have a gripe with this one - it gets rewritten by cp -rf, it's best to come up with a separate empty folder for user overrides like this.

Reply:
You are right though, the sh string comparison will be very lightweight if there are no hidden regressions in sway 'exec'. But there are always ways to go :D